onedal/covariance/incremental_covariance.py

```diff
@@ -46,8 +53,8 @@ class IncrementalEmpiricalCovariance(BaseEmpiricalCovariance):
         Estimated covariance matrix
     """
 
-    def __init__(self, method="dense", bias=False):
-        super().__init__(method, bias)
+    def __init__(self, method="dense", bias=False, assume_centered=False):
```

Comment:
could you please add _reset method to this class and its usage in sklearnex fit method like it is done in IncrementalLinearRegression? otherwise two consecutive fit calls or fit call after partial_fit call give wrong results.